File: MediCareWebSpa/src/app/DTOs/response/login-response.dto.ts
export class LoginResponseDTO {
  userId: number;
  userName: string;
  userSurname: string;
  accessToken: string;
  refreshToken: string;
  roleType: number;

  constructor(
    userId: number,
    userName: string,
    userSurname: string,
    accessToken: string,
    refreshToken: string,
    roleType: number
  ) {
    this.userId = userId;
    this.userName = userName;
    this.userSurname = userSurname;
    this.accessToken = accessToken;
    this.refreshToken = refreshToken;
    this.roleType = roleType;
  }
}

File: MediCareWebApi/DTOs/Response/LoginResponseDTO.cs
using MediCare.Enums;

namespace MediCare.DTOs.Response
{
	public class LoginResponseDTO
	{
		public int UserId { get; set; }
		public string UserName { get; set; }
		public string UserSurname { get; set; }
		public string AccessToken { get; set; }
		public string RefreshToken { get; set; }
		public RoleType RoleType { get; set; }
	}
}
